### PR TITLE
feat(tooltip): Adds offsetSkidding and offsetDistance props for the tooltip

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Adds props to set the offset skidding and distance for the `<tds-tooltip>`

**Solving issue**  
#509 

**Todo**
- [ ] Adds tests (doesn't work great on my local setup)

**How to test**  
1. Go to Tooltip
2. Try out the offsetSkidding and offsetDistance controls.

**Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [ ] All existing tests pass
- [ ] I have updated the documentation (if applicable)

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

### CODE REVIEW

File: packages/core/src/components/tooltip/tooltip.stories.tsx [hunk 1]
```diff
@@ -79,12 +79,26 @@ export default {
...
+      control: {
+        type: 'number',
+      },
+    },
+    offsetSkidding: {
+      name: 'Offset Skidding',
+      control: {
+        type: 'number',
+      },
+    },
```
Comment (theJohnnyMe) [inline packages/core/src/components/tooltip/tooltip.stories.tsx:93]:
```suggestion
    offsetDistance: {
      name: 'Offset Distance',
      description: 'Sets the distance between the wrapped component and Tooltip',
      control: {
        type: 'number',
      },
    },
    offsetSkidding: {
      name: 'Offset Skidding',
      description: 'Sets offset to Tooltip's position, moving Tooltip left-right or top-down - depending on the placement prop.'
      control: {
        type: 'number',
      },
    },
```

Comment (mJarsater) [reply packages/core/src/components/tooltip/tooltip.stories.tsx:93]: I thought the offsetDistance just moved the tooltip vertically and the skidding moved it horizontally? @theJohnnyMe

Comment (theJohnnyMe) [reply packages/core/src/components/tooltip/tooltip.stories.tsx:93]: > I thought the offsetDistance just moved the tooltip vertically and the skidding moved it horizontally? @theJohnnyMe

Try this, it depends on the placment prop:
Screenshot 2024-02-20 at 10 24 35

Comment (theJohnnyMe) [reply packages/core/src/components/tooltip/tooltip.stories.tsx]: offsetDistance is like padding around the wrapped component while skidding is moving the moving Tooltip a bit left/right or top/down, depending on the placement prop it seems.

**Comment by theJohnnyMe** [comment]
@mJarsater  - Please double check if rebase and force push dropped some commits. I can see only story file change. Maybe it is GitHub glitch.

**Comment by mJarsater** [comment]
> @mJarsater - Please double check if rebase and force push dropped some commits. I can see only story file change. Maybe it is GitHub glitch.

hmm, looking at develop my stuff is already there 🤔

**Review by timrombergjakobsson — APPROVED** [review]

**Review by theJohnnyMe — APPROVED** [review]

**Pull request merged**